apollo-router/src/apollo_telemetry.rs

```diff
@@ -319,7 +320,9 @@ impl SpanExporter for Exporter {
                     .to_string();
                 let operation_name = span
                     .attributes
-                    .get(&opentelemetry::Key::from_static_str("operation_name"));
+                    .get(&opentelemetry::Key::from_static_str("operation_name"))
+                    .unwrap_or(&anonymous_or_optional_operation_name);
```

Comment:
I like this. Super safe in case the instrumentation is ever edited or removed...